spec/site/connect_spec.rb

```diff
@@ -51,22 +51,52 @@ def check_sequence version
     end
   end
 
+# Verify that the handshake sequence upon connection is correct for 3.1.1
+#
+# 1. Given the site is new, connected and version 3.1.1
+# 2. Send and receive handshake messages
+# 3. Expect the handshake messages to be in the specified sequence corresponding to version 3.1.1
+# 4. Expect the site to be ready
   it 'exchanges correct connection sequence of rsmp version 3.1.1', rsmp: '3.1.1' do |example|
     check_sequence '3.1.1'
   end
 
+# Verify that the handshake sequence upon connection is correct for 3.1.2
+#
+# 1. Given the site is new, connected and version 3.1.2
```

Comment:
maybe "Given the site is connected and using core 3.1.2"

Reply:
Sounds good, changed it.